src/tribol/geom/ContactPlane.cpp

```diff
@@ -46,29 +48,55 @@ bool CheckInterfacePair( InterfacePair& pair,
          // get instance of contact plane manager to store all contact plane data per cycle
          ContactPlaneManager& cpMgr = ContactPlaneManager::getInstance();
 
+         // set whether full overlap is to be used or not. Note: SINGLE_MORTAR and 
+         // MORTAR_WEIGHTS drop into this 'case', so the method still needs to be checked
          bool full = (cMethod == COMMON_PLANE) ? false : true;
```

Comment:
Minor: can some/all of the vars in this block be `const`?

Reply:
Fixed.